DiamondLightSource/dodal
# Add some basic devices for i04

Fixes #141 also fixes #155

To test:
* Run `python tests/system_tests/test_i03_system.py` - confirm devices all connect
* Run `python tests/system_tests/test_i04_system.py` - confirm devices all connect
